plugins/guests/debian/cap/configure_networks.rb

```diff
@@ -9,8 +10,22 @@ module Cap
       class ConfigureNetworks
         include Vagrant::Util
 
+        @logger = Log4r::Logger.new("vagrant::plugins::guest")
+
         def self.configure_networks(machine, networks)
           machine.communicate.tap do |comm|
+            main_interface = ''
+            comm.execute(%q!ip route | awk '$1=="default"{print $NF;exit}'!) do |type, data|
+              main_interface = data.chomp if type == :stdout
+            end
+            @logger.debug("debian configure_networks. main_interface=#{main_interface.inspect}")
+            available_interfaces = []
+            comm.execute("ls /sys/class/net | grep -v -E '^(lo$|docker|lxc)'") do |type, data|
+              available_interfaces = data.chomp.split("\n") if type == :stdout
+            end
+            available_interfaces.delete(main_interface)
+            @logger.debug("debian configure_networks. available_interfaces=#{available_interfaces.inspect}")
```

Comment:
Do you still need the debug logging?  Should "require log4r" and logger references be removed?